test/freenet/client/filter/m3u/unsafe_madesafe.m3u

```diff
@@ -1,4 +1,4 @@
-http://localhost:8888/external-link/?_CHECKED_HTTP_=http://betray.user/simplepathname.mp3
+#bad-uri-removed
```

Comment:
same here: no longer supported, since the simpler "just remove it" works sufficiently well (and we have no GUI for "let’s check whether the user really wants this" anyway, so this part of the test wasn’t correct, but is now).